report/5.Conclusion/Conclusion.tex

```diff
@@ -1,3 +1,75 @@
 % !TeX root = ../main.tex
+\chapter{Conclusion}
 \section{Conclusion}
+The goal of this project was to create a compiler for a programming language.
+The resulting solution is the programming language HCL and the HCL compiler.
+
+\subsection{Conclusion of the problem-statement}
+HCL is designed with novices in mind, while also implementing high-order functionality, with intuitive syntax and semantics.
+HCL is a hybrid language of the functional and imperative paradigms, with dynamic binding of values.
+HCL has English-like syntax, that also implements high-order functionality.
+%However, as of the conclusion of this report, no user-tests have been conducted (This sentence is really fucking out of place. I have removed it.)
+
+The similarities with the English language is well exemplified in snippet \ref{lis:ConclusionWhile}.
+\begin{lstlisting}[language=hcl,label=lis:ConclusionWhile,caption=English-like syntax example]
+var j = 3
+func subtract = () : none { j = j - 1 }
+func jIsGreaterThan1 = () : bool { 1 lessThan j }
+subtract while jIsGreaterThan1
+\end{lstlisting}
+In snippet \ref{lis:ConclusionWhile}, lines 1 to 3 consists of declarations, line 8 consists of the function-call to While.
+The While function takes two Lambda expressions as arguments, in this case, two function-calls.
+The similarities between the function-call on line 8 and the equivalent English sentence is obvious.
+
+Since user-tests have not been conducted, it cannot be verified whether the English-like syntax allows for easier transition, from English to HCL, for novices.
```

Comment:
keep a hold on those muthafu***g commas
could probly be like: 

Since user-tests have not been conducted, it cannot be verified whether the English-like syntax allows for novices to have an easier transition from English to HCL.

Reply:
"has" not been conducted.right?..